crates/bevy_ui/src/focus.rs

```diff
@@ -25,7 +25,7 @@ use smallvec::SmallVec;
 /// If a UI node has both [`Interaction`] and [`ComputedVisibility`] components,
 /// [`Interaction`] will always be [`Interaction::None`]
 /// when [`ComputedVisibility::is_visible()`] is false.
-/// This ensures that hidden UI nodes are not interactable,
+/// This ensures that hidden UI nodes are not intractable,
```

Comment:
This is "interactable" as in "able to be interacted with" which is not quite a dictionary word but is used very commonly.

Reply:
```suggestion
/// This ensures that hidden UI nodes are not interactable,
```